projects/igniteui-angular/src/lib/combo/combo.component.spec.ts

```diff
@@ -2338,1188 +1031,1591 @@ describe('igxCombo', () => {
+            let dropdown: IgxComboDropDownComponent;
+            configureTestSuite();
+            beforeAll(async(() => {
+                TestBed.configureTestingModule({
+                    declarations: [
+                        IgxComboSampleComponent
+                    ],
+                    imports: [
+                        IgxComboModule,
+                        NoopAnimationsModule,
+                        IgxToggleModule,
+                        ReactiveFormsModule,
+                        FormsModule
+                    ]
+                }).compileComponents();
+            }));
+            beforeEach(fakeAsync(() => {
+                fixture = TestBed.createComponent(IgxComboSampleComponent);
+                fixture.detectChanges();
+                combo = fixture.componentInstance.combo;
+                dropdown = combo.dropdown;
+                input = fixture.debugElement.query(By.css(`.${CSS_CLASS_INPUTGROUP}`));
+            }));
+            it('should properly call dropdown navigatePrev method', fakeAsync(() => {
+                expect(dropdown.focusedItem).toBeFalsy();
+                expect(dropdown.focusedItem).toEqual(null);
+                expect(combo.collapsed).toBeTruthy();
+                combo.toggle();
+                tick();
+                fixture.detectChanges();
+                expect(document.activeElement).toEqual(combo.searchInput.nativeElement);
+                expect(combo.collapsed).toBeFalsy();
+                combo.handleKeyUp(new KeyboardEvent('keyup', UIInteractions.arrowDownEvent));
+                fixture.detectChanges();
+                expect(dropdown.focusedItem).toBeTruthy();
+                expect(dropdown.focusedItem.itemIndex).toEqual(0);
+                expect((combo as any).virtDir.state.startIndex).toEqual(0);
+                dropdown.navigatePrev();
+                tick();
+                fixture.detectChanges();
+                expect(document.activeElement).toEqual(combo.searchInput.nativeElement);
+                combo.handleKeyUp(new KeyboardEvent('keyup', UIInteractions.arrowDownEvent));
+                fixture.detectChanges();
+                expect(dropdown.focusedItem).toBeTruthy();
+                expect(dropdown.focusedItem.itemIndex).toEqual(0);
+                dropdown.navigateNext();
+                tick();
+                fixture.detectChanges();
+                expect(dropdown.focusedItem).toBeTruthy();
+                expect(dropdown.focusedItem.itemIndex).toEqual(1);
+                expect((combo as any).virtDir.state.startIndex).toEqual(0);
+                spyOn(dropdown, 'navigatePrev').and.callThrough();
+                dropdown.navigatePrev();
+                tick();
+                expect(dropdown.focusedItem).toBeTruthy();
+                expect(dropdown.focusedItem.itemIndex).toEqual(0);
+                expect((combo as any).virtDir.state.startIndex).toEqual(0);
```

Comment:
You can use combo.virtualizationState.startIndex insteda of casting combo to any